src/main/java/com/skypro/employee/DepartmentService.java

```diff
@@ -0,0 +1,46 @@
+package com.skypro.employee;
+
+import exception.EmployeeNotFoundException;
+import org.springframework.stereotype.Service;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+
+import static java.util.Comparator.comparingInt;
+import static java.util.stream.Collectors.groupingBy;
+
+@Service
+public class DepartmentService implements DepartmentServiceInterface {
+    private final EmployeeServiceInterfaceImpl employeeService;
```

Comment:
```suggestion
    private final EmployeeServiceInterface employeeService;
```
инъекцию надо делать по интерфейсу